src/scripts.js

```diff
@@ -15,14 +15,28 @@ const weeklyStepsChart = activity.getWeeklyActivityStats('numSteps', date);
 const weeklyStairsChart = activity.getWeeklyActivityStats('flightsOfStairs', date);
 const weeklyMinutesChart = activity.getWeeklyActivityStats('minutesActive', date);
 const weeklyStepsVsFriends = user.findFriendSteps(userData, activityData, date);
+let $grid = $('.grid').packery({itemSelector:'.grid-item', columnWidth: 100});
 
 $(document).ready(function () {
   $('#span__current--date').text(date);
   userHandler();
   hydrationHandler();
   sleepHandler();
   activityHandler();
-})
+});
+
+// let $grid = $('section').draggabilly({
+// })
+
+$grid.find('.draggable').each( function( i, gridItem ) {
+  let draggie = new Draggabilly( gridItem );
+  $grid.packery( 'bindDraggabillyEvents', draggie );
+});
+
+// $('.main__page').packery({
+//   // itemSelector: 'section',
+//   gutter: 15
+// });
```

Comment:
@Garrett-Iannuzzi    The draggability looks great.  I'll open an issue regarding how slow the draggability is so we can keep researching.  Let's be sure to remove these commented out lines when we go through for the final code clean up.